setup.py

```diff
@@ -133,5 +153,10 @@ def get_version():
           'verovio.data.Leipzig': os.listdir('./data/Leipzig'),
           'verovio.data.Petaluma': os.listdir('./data/Petaluma'),
           'verovio.data.text': os.listdir('./data/text'),
-      }
+      },
+      python_requires='>=3.6',
```

Comment:
I don’t think wheels build on Python 2.7? https://github.com/rism-ch/verovio/blob/03c54d0011bb4e3e514cc7d282335a4b1e66b7f6/.github/workflows/python-ci-wheel.yml#L28

Reply:
No, it does not. But if you do `python2.7 -m pip install verovio` it uses the source and builds.

Reply:
```
laurent@laika tmp % python2.7 -m pip install verovio
DEPRECATION: Python 2.7 reached the end of its life on January 1st, 2020. Please upgrade your Python as Python 2.7 is no longer maintained. pip 21.0 will drop support for Python 2.7 in January 2021. More details about Python 2 support in pip can be found at https://pip.pypa.io/en/latest/development/release-process/#python-2-support pip 21.0 will remove support for this functionality.
Defaulting to user installation because normal site-packages is not writeable
Collecting verovio
  Using cached verovio-3.1.0.tar.gz (1.8 MB)
Building wheels for collected packages: verovio
  Building wheel for verovio (setup.py) ... done
  Created wheel for verovio: filename=verovio-3.1.0-cp27-cp27m-macosx_11_2_x86_64.whl size=3912952 sha256=bf8ee53970531fef3ead3c2ebf516efe13663ecc7c0f74964ec45ecefdbf9515
  Stored in directory: /Users/laurent/Library/Caches/pip/wheels/ec/7e/40/ee7dac505e3a7ed8cae842774df813a9f92761b9d79e8edcbe
Successfully built verovio
Installing collected packages: verovio
Successfully installed verovio-3.1.0
laurent@laika tmp % python2.7

WARNING: Python 2.7 is not recommended. 
This version is included in macOS for compatibility with legacy software. 
Future versions of macOS will not include Python 2.7. 
Instead, it is recommended that you transition to using 'python3' from within Terminal.

Python 2.7.16 (default, Dec 21 2020, 23:00:36) 
[GCC Apple LLVM 12.0.0 (clang-1200.0.30.4) [+internal-os, ptrauth-isa=sign+stri on darwin
Type "help", "copyright", "credits" or "license" for more information.
>>> import verovio
>>> tk = verovio.toolkit()
>>> tk.getVersion()
'3.1.0-885a9bd'
>>> 
```

Reply:
So should I change that here?

Reply:
That's what I would do but I don't know what the best practice is. I thought that if the package runs on versions you do not provide a wheel for, you would / can still put that version as supported. But I am not a Python expert, so I leave this up to you.

Reply:
That's what I would do but I don't know what the best practice is. I thought that if the package runs on versions you do not provide a wheel for, you would / can still put these versions as supported. But I am not a Python expert, so I leave this up to you.